oca_dependencies.txt

```diff
@@ -1,2 +1,2 @@
 server-ux
-product-attribute https://github.com/akretion/product-attribute 14.0-mig-product_base_mass_addition
```

Comment:
remove specific branch here when https://github.com/OCA/product-attribute/pull/864 is merged